src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/FormPassTest.php

```diff
@@ -48,9 +48,9 @@ public function testAddTaggedTypes()
             array(),
         ));
 
-        $container->setDefinition('form.extension', $extDefinition);
-        $container->register('my.type1', __CLASS__.'_Type1')->addTag('form.type');
-        $container->register('my.type2', __CLASS__.'_Type2')->addTag('form.type');
+        $container->setDefinition('form.extension', $extDefinition->setPublic(true));
+        $container->register('my.type1', __CLASS__.'_Type1')->addTag('form.type')->setPublic(true);
```

Comment:
Such changes in tests cases aren't always strictly required, right? In most cases the visibility does not matter in those test cases.

Reply:
Correct, I did it in batch to, more fine grained would need more time...

Reply:
Diff now reduced to the minimum.